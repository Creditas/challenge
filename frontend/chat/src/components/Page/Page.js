import React from 'react';
import Paper from '@material-ui/core/Paper';
import Grid from '@material-ui/core/Grid';

import styles from './page.css';

const Page = ({children}) => (
  <div className={styles.wrapper}>
    <Grid container justify="center">
      <Grid item xs={12} md={7}>
        <Paper>
          {children}
        </Paper>
      </Grid>
    </Grid>
  </div>
)

export default Page;
