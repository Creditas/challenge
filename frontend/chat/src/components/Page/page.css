.wrapper {
  background-color: #f1f1f1;
  min-height: 100vh;
}

@media (min-width: 700px) {
  .wrapper {
    padding-top: 50px;
  }
}
