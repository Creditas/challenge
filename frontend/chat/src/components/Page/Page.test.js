import React from 'react';
import { shallow } from 'enzyme';
import Page from './Page';

describe('Page', () => {
  it('should render', () => {
    const rendered = shallow(<Page />);
    expect(rendered).toMatchSnapshot()
  })
})
