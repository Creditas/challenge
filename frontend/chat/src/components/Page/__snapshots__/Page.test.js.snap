// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Page />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <WithStyles(Grid)
        container={true}
        justify="center"
>
        <WithStyles(Grid)
                item={true}
                md={7}
                xs={12}
        >
                <WithStyles(Paper) />
        </WithStyles(Grid)>
</WithStyles(Grid)>,
      "className": "wrapper",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Grid)
          item={true}
          md={7}
          xs={12}
>
          <WithStyles(Paper) />
</WithStyles(Grid)>,
        "container": true,
        "justify": "center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Paper) />,
          "item": true,
          "md": 7,
          "xs": 12,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <WithStyles(Grid)
          container={true}
          justify="center"
>
          <WithStyles(Grid)
                    item={true}
                    md={7}
                    xs={12}
          >
                    <WithStyles(Paper) />
          </WithStyles(Grid)>
</WithStyles(Grid)>,
        "className": "wrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Grid)
            item={true}
            md={7}
            xs={12}
>
            <WithStyles(Paper) />
</WithStyles(Grid)>,
          "container": true,
          "justify": "center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(Paper) />,
            "item": true,
            "md": 7,
            "xs": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
